WTWSMS/history/characters/hunnic.txt

```diff
@@ -255,10 +254,10 @@
 	religion="tengri_pagan"
 	culture="hunnic"
 	trait="strong"
-	trait="wroth"	
+	trait="wroth"
 	trait="brilliant_strategist"
-	320.1.2 = {
-		birth="320.1.2"
+	302.1.2 = {
+		birth=yes
```

Comment:
Why did you modify this?

Reply:
He was too young to be a father. You said Balamber should be edited rather than Baltazar: "As I said, it should be 320, it is rather the date of birth of his father that should be changed.
 "